docs/List.md

```diff
@@ -296,8 +296,9 @@ import {
     useNotify,
     useUnselectAll,
 } from 'react-admin';
+import { VisibilityOff } from '@material-ui/icons';
 
-const ResetViewsButton = ({ selectedIds }) => {
+const ResetViewsButton = ({ resource, selectedIds }) => {
```

Comment:
I'd use 'posts' instead of `resource` un the `unselectAll` call. I find it more explicit.

Reply:
Done.